### ISSUE TITLE
Using Codec[A] with KafkaStreams

### ISSUE THREAD

**Issue by caenrique** [fd4s/fs2-kafka#1144]
I'm thinking about how to get a `org.apache.kafka.common.serialization.Serde` from a Codec.

We are moving our schema from json avro to case classes with vulcan, and we use it with fs2 without problem, but we also have kafka streams applications.

Without including this (fs2-kafka) dependency, I think the only option would be to re-imlement a lot of the functionality to make the schema work well with the schema registry (eg: the vulcan module here but targeting the java Serde)

Basically I would appreciate help with interop between vulcan and native kafka apis :) 
Maybe I'm missing something and there is a simple way. If not it would be awesome to have it
